.github/workflows/generate_minified_version.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
Unfortunately, you can't directly build the minified version:
- the minified file is missing the introductory comments and the `@OnlyCurrentDoc` directive
- in the non-minified file there's the line `User("admin", "PUT_STRONG_PASSWORD_HERE", ALL);` that I want to keep as an example, but this line is hidden at the end of the minified file.

I suggest removing this change from the scope of this PR and discuss this further separately.